apisix/utils/batch-processor.lua

```diff
@@ -146,8 +150,20 @@ function batch_processor:push(entry)
         return
     end
 
+    if not batch_metrics and prometheus.get_prometheus() and self.name
+       and self.route_id and self.server_addr then
+        batch_metrics = prometheus.get_prometheus():gauge("batch_process_entries",
+                                                          "batch process remaining entries",
```

Comment:
We need to update the https://github.com/apache/apisix/blob/master/doc/plugins/prometheus.md#available-metrics

Reply:
added.